src/opsinputs/opsinputs_fill_mod.F90

```diff
@@ -2279,13 +2299,30 @@ function opsinputs_fill_varnames_with_channels(VarName, Channels) result(VarName
 character(len=max_varname_with_channel_length) :: VarNames(max(size(Channels), 1))
 integer                                        :: ichan
 
-if (size(Channels) == 0) then
-  VarNames(1) = VarName
-else
-  do ichan = 1, size(Channels)
-    write (VarNames(ichan),'(A,"_",I0)') VarName, Channels(ichan)
-  end do
-end if
+!character(len=max_varname_with_channel_length) :: VarNames_emis(max(size(Channels), 1))
+
+
+
+!if (Varname=="emissivity") then
+!  do ichan = 1, size(Channels)
+!    write(*,*) "here"
+!    write(*,*) VarName, Channels(ichan)
+!    write (VarNames_emis(ichan),'(A,"_",I0)') VarName, Channels(ichan)
+   ! write(*,'(A,"_",I0)') VarName, Channels(ichan)
+   ! write (VarNames_emis(ichan),'(A,"_",I0)') VarName, Channels(ichan)
+  !end do
+ ! VarNames = VarNames_emis
+!else
```

Comment:
Can't remember why I have commented this out but also there are no ctest failures?

Reply:
It would be good to understand this.